app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -17,7 +17,7 @@ def forward_request(original_request, controller_name, branch_id, certs)
       logger.debug("Sending request to: #{request_opts[:url]}")
 
       execute_cloud_request(request_opts)
-    rescue RestClient::Exception => error_response
+    rescue RestClient::ExceptionWithResponse => error_response
```

Comment:
Sounds good, but we need to take into account the proper reporting of errors without a response.

I suppose we will need to modify the reporting block: https://github.com/theforeman/foreman_rh_cloud/blob/d642c75dc2b2701ba9ab412e09dfefd9778f2c0e/app/controllers/insights_cloud/api/machine_telemetries_controller.rb#L28

Reply:
Is that so?

That block is the whole reason I came here, as when you get a `RestClient::SSLCertificateNotVerified`, that block still calls `.code` but on `nil` (as `error_response.response` that we return here is `nil`)